### PR TITLE
RHDEVDOCS-2755 - API Ref Draft

### PR DESCRIPTION
**Pull request by libander**
Version(s):

Issue: https://issues.redhat.com/browse/RHDEVDOCS-2755 

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/logging-api-reference.adoc [hunk 1]
```diff
@@ -0,0 +1,1155 @@
...
+===== Description
+InputSpec defines a selector of log messages.
+
+=====  Type
+* array
+
+[options="header"]
+|======================
+|Property|Type|Description
+|application|object|  *(optional)* Application, if present, enables named set of `application` logs that
```
Comment (libander) [inline modules/logging-api-reference.adoc:47]: @jcantrill - 

> enables named set of `application` logs that .... 

Seems to have gotten truncated, what should this say?

File: modules/logging-api-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,1155 @@
...
+//Needs description
+=====  Type
+* object
+
+[options="header"]
+|======================
+|Property|Type|Description
+|enableStructuredContainerLogs|bool|  *(optional)* EnableStructuredContainerLogs enables multi-container structured logs to allow
+|structuredTypeKey|string|  *(optional)* StructuredTypeKey specifies the metadata key to be used as name of elasticsearch index
+|structuredTypeName|string|  *(optional)* StructuredTypeName specifies the name of elasticsearch schema
```
Comment (libander) [inline modules/logging-api-reference.adoc:125]: > |enableStructuredContainerLogs|bool|  *(optional)* EnableStructuredContainerLogs enables multi-container structured logs to allow
|structuredTypeKey|string|  *(optional)* StructuredTypeKey specifies the metadata key to be used as name of elasticsearch index
|structuredTypeName|string|  *(optional)* StructuredTypeName specifies the name of elasticsearch schema

---
Truncated.

File: modules/logging-api-reference.adoc [hunk 3]
```diff
@@ -0,0 +1,1155 @@
...
+===== Description
+PipelinesSpec link a set of inputs to a set of outputs.
+
+=====  Type
+* array
+
+[options="header"]
+|======================
+|Property|Type|Description
+|detectMultilineErrors|bool|  *(optional)* DetectMultilineErrors enables multiline error detection of container logs
```
Comment (libander) [inline modules/logging-api-reference.adoc:200]: Truncated?

File: modules/logging-api-reference.adoc [hunk 4]
```diff
@@ -0,0 +1,1155 @@
...
+|inputRefs|array|  InputRefs lists the names (`input.name`) of inputs to this pipeline.
+|labels|object|  *(optional)* Labels applied to log records passing through this pipeline.
+|name|string|  *(optional)* Name is optional, but must be unique in the `pipelines` list if provided.
+|outputRefs|array|  OutputRefs lists the names (`output.name`) of outputs from this pipeline.
+|parse|string|  *(optional)* Parse enables parsing of log entries into structured logs
```
Comment (libander) [inline modules/logging-api-reference.adoc:205]: > |parse|string|  *(optional)* Parse enables parsing of log entries into structured logs 

Truncated?

File: modules/logging-api-reference.adoc [hunk 5]
```diff
@@ -0,0 +1,1155 @@
...
+=====  Type
+* object
+
+[options="header"]
+|======================
+|Property|Type|Description
+|collection|object|  Specification of the Collection component for the cluster
+|curation|object| **(DEPRECATED)** *(optional)* Deprecated. Specification of the Curation component for the cluster
+|forwarder|object| **(DEPRECATED)** *(optional)* Deprecated. Specification for Forwarder component for the cluster
+|logStore|object|  *(optional)* Specification of the Log Storage component for the cluster
```
Comment (libander) [inline modules/logging-api-reference.adoc:255]: > |collection|object|  Specification of the Collection component for the cluster
> |curation|object| **(DEPRECATED)** *(optional)* Deprecated. Specification of the Curation component for the cluster
> |forwarder|object| **(DEPRECATED)** *(optional)* Deprecated. Specification for Forwarder component for the cluster
> |logStore|object|  *(optional)* Specification of the Log Storage component for the cluster

Truncated?

File: modules/logging-api-reference.adoc [hunk 6]
```diff
@@ -0,0 +1,1155 @@
...
+=====  Type
+* array
+
+[options="header"]
+|======================
+|Property|Type|Description
+|effect|string|  *(optional)* Effect indicates the taint effect to match. Empty means match all taint effects.
+|key|string|  *(optional)* Key is the taint key that the toleration applies to. Empty means match all taint keys.
+|operator|string|  *(optional)* Operator represents a key&#39;s relationship to the value.
+|tolerationSeconds|int|  *(optional)* TolerationSeconds represents the period of time the toleration (which must be
```
Comment (libander) [inline modules/logging-api-reference.adoc:858]: > |tolerationSeconds|int|  *(optional)* TolerationSeconds represents the period of time the toleration (which must be 

Truncated

File: modules/logging-api-reference.adoc (hunk 1 above)
Comment (jcantrill) [reply modules/logging-api-reference.adoc:47]: I don't know if there is a character limit or a break in the note that would keep us from getting the entire content

**Comment by libander** [comment]
@kabirbhartiRH / @QiaolingTang / @anpingli - Review please.

**Comment by rolfedh** [comment]
@mburke5678 FYI

**Pull request closed without merging**